src/Common/src/TypeSystem/Common/TypeDesc.cs

```diff
@@ -456,5 +456,31 @@ public virtual MethodDesc GetFinalizer()
         {
             return null;
         }
+
+        public bool RequiresAlign8
```

Comment:
I do not think this belongs to TypeDesc. It is pretty specific to how EEType works. It should be a method somewhere in the Runtime/EEType-specific area.